dev/localstack/docker-compose.yml

```diff
@@ -15,3 +15,5 @@ services:
     volumes:
       - "${LOCALSTACK_VOLUME_DIR:-./volume}:/var/lib/localstack"
       - ./init:/etc/localstack/init/ready.d
+    networks:
+    - qontract-development
```

Comment:
```bash
$ make localstack
service "localstack" refers to undefined network qontract-development: invalid compose project
make: *** [localstack] Error 15
```

maybe do it together on usage side? https://docs.docker.com/compose/multiple-compose-files/include/#include-and-overrides

Reply:
Yeah, this does the trick! 👍